### ISSUE TITLE
Pull request check incorrectly marking valid commit messages as invalid

### ISSUE THREAD

**Issue by nzakas** [eslint/eslint#5340]
It looks like the commit message check is not exactly correct on pull requests. Right now, it incorrectly assumes that the commit message is just one line, so this will trigger the max length warning and the warning about a missing issue reference:

```
Fix: Description (fixes #1234)

Fixed something bad
```

The solution is to only take the first line of the commit message and validate that, leaving the rest alone.
